plugins/MOAction/MOAction.json

```diff
@@ -3,5 +3,5 @@
 	"Name": "Mouseover Action",
 	"Description": "This plugin allows for actions to be used on mouseover targets without a macro.",
 	"InternalName": "MOAction",
-	"AssemblyVersion": "1.0.0.0"
+	"AssemblyVersion": "1.0.0.1"
```

Comment:
Can you add the ApplicableGameVersion in your source files?

Reply:
Added.